.github/workflows/publish.yml

```diff
@@ -12,15 +12,21 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       publish: ${{ steps.check.read-status.publish }}
+    strategy:
+      matrix:
+        node-version: [20]
```

Comment:
who go backwards in node version? 21.x -> 20?

Reply:
Going off of the recommended CI versions: https://pnpm.io/continuous-integration#github-actions